src/components/app-root/app-root.tsx

```diff
@@ -92,7 +87,7 @@ export class AppRoot {
               <stencil-route url="/activity" component="page-activity" />
```

Comment:
let's link the activities pages to their delivery locations

Reply:
or I guess we could just remove this page and redirect `/activity` to `/deliveries`

Reply:
I linked to the activity page from the "view more" link in the previous deliveries card. Not doing any location filtering though. Thoughts?